FlightPlanProject/Tools/SettingsFile.cs

```diff
@@ -0,0 +1,307 @@
+
+using UnityEngine;
```

Comment:
this is the class used for loading and writting settings values. it uses a settings.json file a the root of the plugin folder